### PR TITLE
[NNAPI]Fix error in axis of reduce_mean.

### PR DESCRIPTION
**Pull request by H1Gdev**
1. create TensorFlow model.
```python
tf.reduce_mean(x, 0)
```
1. convert to TensorFlow lite model.
1. execute in NNAPI delegate.
1. error occurred.
   `E/tflite: NN API returned error ANEURALNETWORKS_BAD_DATA at line 1465 while setting new operand value from memory for tensor XXX.`

### CODE REVIEW

**Review by miaowang14 — APPROVED** [review]
Thanks for the fix!
